castai/add-castai-removal-disabled/artifacthub-pkg.yml

```diff
@@ -0,0 +1,23 @@
+name: add-castai-removal-disabled
+version: 1.0.0
+displayName: Add CAST AI removal disabled
+createdAt: "2023-04-10T20:11:12.000Z"
+description: >-
+  CAST AI will not downscale a node that includes a pod with the autoscaling.cast.ai/removal-disabled="true" label on it, this protects sensitive workloads from being evicted and can be attributed to any pod to protect against unwanted downscaling. This policy will mutate jobs and cronjobs to add the removal-disabled label to protect against eviction. 
+install: |-
+  ```shell
+  kubectl apply -f https://raw.githubusercontent.com/kyverno/policies/main/castai/add-castai-removal-disabled/add-castai-removal-disabled.yaml
+  ```
+keywords:
+  - kyverno
+  - CAST AI
+  - EKS Best Practices
+readme: |
+  CAST AI will not downscale a node that includes a pod with the autoscaling.cast.ai/removal-disabled="true" label on it, this protects sensitive workloads from being evicted and can be attributed to any pod to protect against unwanted downscaling. This policy will mutate jobs and cronjobs to add the removal-disabled label to protect against eviction. 
+  
+  Refer to the documentation for more details on Kyverno annotations: https://artifacthub.io/docs/topics/annotations/kyverno/
+annotations:
+  kyverno/category: "CAST AI, EKS Best Practices"
+  kyverno/kubernetesVersion: "1.23"
+  kyverno/subject: "Pod"
```

Comment:
Value here is "Job, CronJob" since that's what you're matching on, not Pod

Reply:
Updated